packages/pg-v5/lib/util.js

```diff
@@ -149,3 +150,24 @@ exports.databaseNameFromUrl = (uri, config) => {
   uri = url.parse(uri)
   return `${uri.hostname}:${uri.port || 5432}${uri.path}`
 }
+
+exports.parsePostgresConnectionString = (db) => {
+  const url = require('url')
```

Comment:
Any reason/pattern why we're requiring modules inside functions?

Reply:
It predates me – I'm not personally a fan but keeps it consistent with the rest of `util`. I'd like to heavily refactor a bunch of this stuff but ⏳

Reply:
Understood!

Reply:
we require inside functions to lazy-load modules for performance reasons. It's not necessary with oclif plugins, but this is an older plugin which will still benefit from lazy-loading.

Reply:
Thanks, @jdxcode